src/Value.js

```diff
@@ -5,7 +5,10 @@ var Option = React.createClass({
 	displayName: 'Value',
 
 	propTypes: {
-		label: React.PropTypes.string.isRequired
+		label: React.PropTypes.string.isRequired,
+		onRemove: React.PropTypes.func,
+		optionLabelClick: React.PropTypes.bool,
+		onOptionLabelClick: React.PropTypes.func
```

Comment:
This might have been worth pulling out to a separate PR.  Could have been merged sooner :).